README.md

```diff
@@ -154,6 +154,13 @@ These error messages can also be amended via the Adapt Authoring Tool - but must
 "CLIENT_COULD_NOT_CONNECT": "La connexion avec la plate-forme de formation n'a pas pu être établie."
 ```
 
+### Print completion information from LMS data
+Using the lms suspend data from a different course:
+```js
+var suspendData  = "{\"lang\":\"en\",\"a11y\":false,\"captions\":\"en\",\"c\":\"hAA\",\"q\":\"oAPQ4XADAcATDAHC4EYDgCYYA4XEDAcATDAHC4kYDgCYYA4XIDAcATDAHC5gbDgCYYbjhdAMBwBAMAcLqBgOAIBgDhdgMBwBAMAcLuBgOFwBABgDhdyMBwBAMAcLQgDAcLgCADAHC0JAwHC4AgAwBwtCSMBwBAMAcLQoDAcLgCADAHC0LAwHC4AgAwBwtDAMBwuAIAMAcLQwjAcAQDAHC0NAwHC4AgAwBwtDSMBwBAMAcLQ4DAcLgCADAHC0OIwHAEAwBwtDwMBwuAIAMAcLQ8jAcAQDAHC0QAwHC4AgAwBwtECMBwBAMAcLREDAcLgCADAHC0RIwHAEAwBwtEgMBwBAMAcLRMDAcAQDAA\"}";
+require('core/js/adapt').spoor.statefulSession.printCompletionInformation(suspendData);
+```
+
```

Comment:
```suggestion
That will output something like the following:
```console
INFO: course._isComplete: false, course._isAssessmentPassed: false, block completion: 11110000000000000000
```
Which, in the above example, indicates that the learner only completed the blocks with trackingIds 0, 1, 2, & 3.

Reply:
The above sentence 'Which, in the above example,...' should be part of the suggestion but I can't seem to find a way to make it accept a suggestion that includes closing backticks!!

Reply:
Sorted.

Reply:
Genius! thank you